static/src/javascripts/projects/common/utils/$.js

```diff
@@ -8,6 +8,13 @@ define([
     _
 ) {
 
+    // Warning: side effect. This patches the bonzo module for use everywhere
+    bonzo.aug({
+        height: function () {
+            return this.dim().height;
+        }
+    });
+
```

Comment:
Not sure about this. I would prefer to extend the constructor/“class”, but bonzo unfortunately doesn't expose the constructor—instead they have this `bonzo.aug`.